### DISCUSSION TITLE
. ./activate command doesnt work while installing

### DISCUSSION THREAD

**Discussion by jimenezfer** [Chia-Network/chia-blockchain#4723] [Support]
x86 ubuntu 20.04 
/home/user/chia/chia-blockchain (dir where i execute . ./activate)
Python 3.8

i get this:

bash: ./activate: No such file or directory

am in here in the install guide:
https://github.com/Chia-Network/chia-blockchain/wiki/INSTALL#ubuntudebian

How do i activate  ?

To run this next: sh install-gui.sh

Thanks!

**Comment by ilyakooo0** [comment]
Could you please describe all of the steps you did?

**Reply by jimenezfer** [reply]
yes,

# Checkout the source and install
1- git clone https://github.com/Chia-Network/chia-blockchain.git -b latest --recurse-submodules
2- cd chia-blockchain

3- sh install.sh

4- . ./activate (fails )

# The GUI requires you have Ubuntu Desktop or a similar windowing system installed.
# You can not install and run the GUI as root

5- sh install-gui.sh (cant do this one am stucked. :( )

**Reply by jimenezfer** [reply]
Thank you!

**Reply by lvcivs** [reply]
Do you have the file with the name `activate` in that dir? If not, something went wrong with your install.

**Reply by jimenezfer** [reply]
No i dont.

**Reply by jimenezfer** [reply]
ill retry the install.

**Reply by jimenezfer** [reply]
same, no activate file

**Reply by jimenezfer** [reply]
this is the output of the command: sh install.sh

**Reply by jimenezfer** [reply]
![image](https://user-images.githubusercontent.com/23113643/118043839-e751a380-b343-11eb-8626-ab2c67f8e330.png)

**Comment by lvcivs** [comment]
Yeah, looks like `apt` aborted for some reason. Maybe a problem with your system clock. Have a look at this: https://askubuntu.com/questions/1096930/sudo-apt-update-error-release-file-is-not-yet-valid

**Reply by jimenezfer** [reply]
**Error: Command '['/home/user/chia/chia-blockchain/venv/bin/python3.7', '-Im', 'ensurepip', '--upgrade', '--default-pip']' returned non-zero exit status 1.**

**Reply by jimenezfer** [reply]
that was correct auto time setup was not on, i turn it on and retried.

**Comment by l-we** [comment]
sh bash zsh fish csh ....

**Reply by jimenezfer** [reply]
i figure it out. 

I just ran "apt-get install python3.7-dev python3.7-venv" 

i was missing the venv lib for python 3.7

**Comment by jimenezfer** [comment]
Now i just need to make this thing sync, to be able to plot and farm.

That was the reason to reinstall.

Thank you !!

**Comment by Nuyoah66** [comment]
before run command：`./activate`   you must run this command：`chmod +x venv/bin/activate`

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
